src/transaction/CreateTransaction.scss

```diff
@@ -87,5 +87,79 @@
             align-self: center;
         }
     }
+
+    // Source: https://stackoverflow.com/questions/11679567/using-css-for-a-fade-in-effect-on-page-load
+    .fadeIn {
+      -webkit-animation: fadein .5s; /* Safari, Chrome and Opera > 12.1 */
+       -moz-animation: fadein .5s; /* Firefox < 16 */
+        -ms-animation: fadein .5s; /* Internet Explorer */
+         -o-animation: fadein .5s; /* Opera < 12.1 */
+            animation: fadein .5s;
+    }
+
+    @keyframes fadein {
+        from { opacity: 0; }
+        to   { opacity: 1; }
+    }
+
+    /* Firefox < 16 */
+    @-moz-keyframes fadein {
+        from { opacity: 0; }
+        to   { opacity: 1; }
+    }
+
+    /* Safari, Chrome and Opera > 12.1 */
+    @-webkit-keyframes fadein {
+        from { opacity: 0; }
+        to   { opacity: 1; }
+    }
+
+    /* Internet Explorer */
+    @-ms-keyframes fadein {
+        from { opacity: 0; }
+        to   { opacity: 1; }
+    }
+
+    /* Opera < 12.1 */
+    @-o-keyframes fadein {
+        from { opacity: 0; }
+        to   { opacity: 1; }
+    }
+
+    .fadeOut {
+      -webkit-animation: fadeOut .5s; /* Safari, Chrome and Opera > 12.1 */
+       -moz-animation: fadeOut .5s; /* Firefox < 16 */
+        -ms-animation: fadeOut .5s; /* Internet Explorer */
+         -o-animation: fadeOut .5s; /* Opera < 12.1 */
+            animation: fadeOut .5s;
+    }
+
+    @keyframes fadeOut {
+      from {opacity: 1;}
+      to {opacity: 0;}
+    }
+    /* Firefox < 16 */
+    @-moz-keyframes fadeOut {
+      from { opacity: 1; }
+      to   { opacity: 0; }
+    }
+
+    /* Safari, Chrome and Opera > 12.1 */
+    @-webkit-keyframes fadeOut {
+        from { opacity: 1; }
+        to   { opacity: 0; }
+    }
+
+    /* Internet Explorer */
+    @-ms-keyframes fadeOut {
+        from { opacity: 1; }
+        to   { opacity: 0; }
+    }
+
+    /* Opera < 12.1 */
+    @-o-keyframes fadeOut {
+        from { opacity: 1; }
+        to   { opacity: 0; }
+    }
```

Comment:
Do not re-create fades yourself (and if you do, please write it yourself - for instance, the browser prefixes are now unnecessary).

See https://getbootstrap.com/docs/5.0/components/toasts/#live instead

Reply:
Removed. I've tried things in the link, but I will figure out a better way to go about this